app/controllers/projects_controller.rb

```diff
@@ -73,6 +76,24 @@ def destroy
     end
   end
 
+  def add_owner
```

Comment:
long controller methods like this are a real red flag - not sure why codeclimate isn't picking this up - as I was saying in other PRs we are really looking for logic to be pushed out of views/controllers and into models and services